Automattic/wp-calypso
# Signup header banner with refund policy in plans step - clean up experiment (keep control)

Related to https://github.com/Automattic/martech/issues/2790

## Proposed Changes

Follow up to https://github.com/Automattic/wp-calypso/pull/89601 and https://github.com/Automattic/wp-calypso/pull/89602 to clean up the experiment. Removes the banner with the 14-day refund policy in favor of control. See pau2Xa-5Ty-p2#comment-15674

**Notes:** I debated whether to strip the whole thing out or to keep the banner logic and just not render it. The implementation is decent and can be used for any other case where something needs to sit in the overlay banner. It's also configurable to be of white/dark background. I have seen designs where something is indeed mentioned in a header banner as we did here, although until those materialize the banner component would be dead code. We can always refer back to this PR and reimplement it if the need arises.

## Media

## Why are these changes being made?


Clean up experiment code.

## Testing Instructions



* Go to `/start/plans` and confirm no blue overlay banner with 14-day refund policy mention renders in the header

## Pre-merge Checklist
